packages/cli/src/databases/migrations/postgresdb/1692967111174-ChangeVersionIdToUUID.ts

```diff
@@ -0,0 +1,9 @@
+import type { IrreversibleMigration, MigrationContext } from '@/databases/types';
+
+export class ChangeVersionIdToUUID1692967111174 implements IrreversibleMigration {
+	async up({ queryRunner, tablePrefix }: MigrationContext) {
+		await queryRunner.query(
+			`ALTER TABLE "${tablePrefix}workflow_entity" ALTER "versionId" TYPE uuid USING "versionId"::uuid`,
```

Comment:
do we need this migration in this PR? if not, I'd prefer this to be in a separate PR whenever we need it.